CHANGELOG.md

```diff
@@ -1,5 +1,34 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2022-09-04
+
+Monitor the release status by regions at [AKS-Release-Tracker](http://aka.ms/aks/release-tracker).
+
+### Announcements
+* The memory limits of liveness-probe container and node-driver-registrar container running in AzureDisk and AzureFile pods on Windows nodes are increased from 100MiB to 150MiB.
+* Pod Disruption Budget (PDB) improvement: 
+  * AKS node drain failures will have better error messages with more details about PDB and unready pods. 
+  * Upgrade will fail sooner when there is a Pod Disruption Budget with max unavailable == 0.
+* Add VMSS tag and node label ebpf-dataplane=cilium when Cilium is enabled
+* Starting with Kubernetes 1.25, the following changes will be made default:
+  * Ubuntu 22.04 for x86, AMD and ARM64 architectures will be the default host.
+  * Windows Server 2022 will be the default Windows host. Important, old windows 2019 containers will not work on windows server 2022 hosts.
+* Kubernetes 1.21 version has been deprecated as of July 31st, 2022. See [documentation](https://docs.microsoft.com/azure/aks/upgrade-cluster?tabs=azure-cli) on how to upgrade your cluster.
```

Comment:
Do we need this now?

Reply:
Yes we leave these here

Reply:
Sorry. I meant the 1.21 depreciation as it was over a month ago.